code/modules/mob/grab/normal/grab_normal.dm

```diff
@@ -43,14 +43,18 @@
 		return FALSE
 
 	var/mob/living/assailant = G.assailant
-	if(!assailant || !assailant.skill_check(SKILL_COMBAT, SKILL_ADEPT))
+	if(!assailant)
 		return FALSE
 
 	var/obj/item/organ/external/O = G.get_targeted_organ()
 	if(!O)
 		to_chat(assailant, SPAN_WARNING("\The [affecting] is missing that body part!"))
 		return FALSE
 
+	if(!assailant.skill_check(SKILL_COMBAT, SKILL_ADEPT))
+		to_chat(assailant, SPAN_WARNING("You attempt clumsily but struggle to do a proper jointlock with \the [affecting]'s [O.name]!"))
```

Comment:
Would suggest:
`You clumsily attempt to jointlock \the [affecting]'s [O.name]. but fail!`

Reply:
You're right, that does sounds a lot better XD